bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,837 +4,511 @@
+				<option value="PERSON_DEPARTURE">Person has left</option>
+				<option value="PERSON_SEEN">Person has been seen</option>
+				<option value="MOVEMENT_HUMAN">Human seen</option>
+				<option value="MOVEMENT_VEHICLE">Car seen</option>
+				<option value="MOVEMENT_ANIMAL">Animal seen</option>
+			</options>
+		</state>
 	</channel-type>
 
-	<channel-type id="maxHumidityThisWeek" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Max Humidity This Week</label>
-		<description>Minimum Humidity this week</description>
-		<category>Humidity</category>
-		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	<channel-type id="home-event">
+		<kind>trigger</kind>
+		<label>Home Event</label>
+		<event>
+			<options>
+				<option value="PERSON"/>
+				<option value="PERSON_AWAY"/>
+				<option value="OUTDOOR"/>
+				<option value="MOVEMENT"/>
+				<option value="NEW_MODULE"/>
+				<option value="MODULE_CONNECT"/>
+				<option value="MODULE_DISCONNECT"/>
+				<option value="MODULE_LOW_BATTERY"/>
+				<option value="MODULE_END_UPDATE"/>
+				<option value="CONNECTION"/>
+				<option value="DISCONNECTION"/>
+				<option value="ON"/>
+				<option value="OFF"/>
+				<option value="BOOT"/>
+				<option value="SD"/>
+				<option value="ALIM"/>
+			</options>
+		</event>
 	</channel-type>
 
-	<channel-type id="maxHumidityThisMonth" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Max Humidity This Month</label>
-		<description>Maximum Humidity this month</description>
-		<category>Humidity</category>
-		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	<channel-type id="rssi" advanced="true">
+		<item-type>Number:Power</item-type>
+		<label>Signal</label>
+		<description>Signal strength indicator.</description>
+		<category>QualityOfService</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
+		</tags>
+		<state readOnly="true" pattern="%d %unit%"></state>
 	</channel-type>
 
-	<channel-type id="humidex">
-		<item-type>Number</item-type>
-		<label>Humidex</label>
-		<description>Computed Humidex index</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.0f"/>
+	<channel-type id="absolute-pressure" advanced="true">
+		<item-type>Number:Pressure</item-type>
+		<label>Absolute Pressure</label>
+		<description>Pressure measured relative to a full vacuum.</description>
+		<category>Pressure</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Pressure</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f %unit%"/>
 	</channel-type>
 
-	<channel-type id="heatIndex">
+	<channel-type id="heat-index">
 		<item-type>Number:Temperature</item-type>
 		<label>Heat Index</label>
-		<description>Computed Heat Index</description>
+		<description>Apparent computed temperature (based on temperature and humidity).</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
```

Comment:
These tags should be removed as it's a computed temperature, not a measured one.